### DISCUSSION TITLE
Explorer submission guideline for Hedera network

### DISCUSSION THREAD

**Discussion by soharang** [WalletConnect/walletconnect-monorepo#4461] [Developer Support]
Hello, 

I am preparing Explorer submission for my wallet and I wonder if there is other test cases for Hedera network.

As far as I understand, there is only one wallet guideline as below:
https://docs.walletconnect.com/cloud/explorer-submission#how-do-we-test-wallets
and some cases are not for Hedera  - supproting eth_personalSign, supporting eth_signTypedData_v4 and that.

**Comment by glitch-txs** [comment]
Hi, it's been updated, some of them are only for EVM wallets

**Reply by soharang** [reply]
Thanks for the update.
Is there any plan to add Chain Specific cases for Hedera?

**Reply by glitch-txs** [reply]
Yes! If you have any suggestions please let me know

**Reply by soharang** [reply]
Thanks, I raised a related ticket on Hedera WalletConnect's repo.
https://github.com/hashgraph/hedera-wallet-connect/issues/120
